api/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/profclems/compozify/api/router"
+	"log"
+	"net/http"
```

Comment:
nit: let's reorganize this into:

```
<standard imports>

<thirdparty imports>

<project imports>
```

```suggestion
	"log"
	"net/http"
	
	"github.com/gorilla/mux"
	
	"github.com/profclems/compozify/api/router"
```